### DISCUSSION TITLE
Missing documentation for many configuration values.

### DISCUSSION THREAD

**Discussion by saiichihashimoto** [repository-settings/app#532] [Q&A] [answered]
Often, an issue arises like [this one](https://github.com/probot/settings/issues/402#issuecomment-901684292) where someone asks for support of a new config value, but it turns out it's already there. Is this something we can fix as an issue across the board? I'm wondering which of these is true:

1. `probot/settings` is being updated to support these values but we are forgetting to update the README. If so, totally understandable, not much to do here.
2. These config values are actually coming from an API so any new values are actually natively supported. If so, we should reference that API in the documentation, so we can know that it's an ongoing list, and have a link to that list.

**Comment by travi** [comment, marked as answer]
> 2. These config values are actually coming from an API so any new values are actually natively supported.

this is mostly the case and i already have a branch in progress to make this more clear and direct users to the API documentation more clearly. i just need to find the time to finish that effort. there are some cases where the api details dont map cleanly and the setting app takes steps to smooth those differences and those will be explained more clearly as well.

there are already links to the api docs in the example, but it has become more and more clear how often those are overlooked or not understood.

**Reply by saiichihashimoto** [reply]
Ah, I see the link now. I was going to suggest that since I hadn't seen it! Goes to show how it could be more clear like you're saying. Keep on keeping on, thank you!

**Reply by electriquo** [reply]
in the meantime, you might wish to take a look at https://github.com/probot/settings/pull/480
